mabl snippets/GetDateComponents.js

```diff
@@ -0,0 +1,184 @@
+/**
+Returns an object containing values assosiated todays date.
+IE compatiable: true
+
+To access these options, save the output of this snippet to a variable (I'll call 
+it "date" for the example) and then use this syntax: 
+    {{@<variable name>.<option>}}
+like this:
+    {{@date.monthNumeric}}
+
+OPTIONS:
+- dayNumeric
+- dayTwoDigit
+- monthNumeric
+- monthTwoDigit
+- monthNarrow
+- monthShort
+- monthLong
+- yearNumeric
+- yearTwoDigit
+- hour12TwoDigit
+- hour24TwoDigit
+- hour12Numeric
+- hour24Numeric
+- dayPeriod
+- minute
+- second
+ */
+
+function mablJavaScriptStep(mablInputs, callback) {
+  var today = new Date();
+  // Today
+  callback(getDateComponentsFor(today));
+
+  // Tomorrow
+  // callback(getDateComponentsFor(today.addDays(1)));
+
+  // Yesterday
+  // callback(getDateComponentsFor(today.addDays(-1)));
+
+  // Two weeks from now
+  // callback(getDateComponentsFor(today.addDays(14)));
+
+  // Two weeks ago
+  // callback(getDateComponentsFor(today.addDays(-14)));
```

Comment:
It would be nice to add examples for formatting using

* different time zone or UTC
* different locales (or using the locale of the browser)

Reply:
I love this idea, but I think that is for a future version.